Automattic/wc-calypso-bridge
# Update product task to detect modified products

### Changes proposed in this Pull Request:



Closes https://github.com/Automattic/wc-calypso-bridge/issues/973.

Update our custom logic to detect when users have updated the sample products so we don't leave the task as incomplete.




### How to test the changes in this Pull Request:


1. Use a free trial site where the product task is not completed yet.
2. Checkout this branch
3. Go to `WooCommerce > Home`
4. Confirm product task is not completed
5. Go to Products
6. Edit any product
7. Go to `WooCommerce > Home`
8. Confirm product task is completed
